REFERENCE.md

```diff
@@ -0,0 +1,310 @@
+# Reference
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Table of Contents
+
+**Resource types**
+
+* [`ini_setting`](#ini_setting): An arbitrary name used as the identity of the resource.
+* [`ini_subsetting`](#ini_subsetting): An arbitrary name used as the identity of the resource.
+
+**Functions**
+
+* [`create_ini_settings`](#create_ini_settings): Uses create_resources to create a set of ini_setting resources from a hash:      $settings = {  section1 => {         setting1 => val1       
+
+## Resource types
+
+### ini_setting
+
+An arbitrary name used as the identity of the resource.
```

Comment:
This is the description for the `name` param. You will need to add a general description for ini_setting